releasing/cloudbuild_kustomize_image.yaml

```diff
@@ -9,14 +9,25 @@ steps:
       - "PROJECT_ID=$PROJECT_ID"
       - "_GIT_TAG=$_GIT_TAG"
       - "_PULL_BASE_REF=$_PULL_BASE_REF"
-# We need to use bash to configure the build date and version properly.
+  # We need to use bash to configure the build date and version properly.
+  - name: "gcr.io/cloud-builders/docker"
```

Comment:
Maybe `""gcr.io/cloud-builders/docker"` image does not contain `buildx` plugin.
https://github.com/GoogleCloudPlatform/cloud-builders/blob/7b3987f0b23841ce782cf6547ab89ce2996ce983/docker/Dockerfile-versioned#L5-L9